### DISCUSSION TITLE
would it be possible to use ASReview on the general index?

### DISCUSSION THREAD

**Discussion by bozo32** [asreview/asreview#1009] [Q&A]
I support reviews for PhDs who are stuck in very messy disciplines where indexed fields are a lousy predictor of any (let alone relevant) content. While we’ve thought about the limits to databases, the alternatives have their own problems.

[Searching for evidence or approval? A commentary on database search in systematic reviews and alternative information retrieval methodologies](https://pubmed.ncbi.nlm.nih.gov/29106042/) is a bit of a meditation on the limits of abstracts in indexing academic output.

since downloading literally thousands of records has both been time consuming and caused my library apoplexy before
Would asreview be able to use [the general index?](https://archive.org/details/GeneralIndex)

to test the quality of the record identification strategies used in published reviews? (This might be quite ugly. I would dearly like to test the hypothesis that the performance of title/abstract/keywords in predicting content varies across disciplines)
And/Or
Given an appropriate manually coded training set generated through typical database retrieval methods
Use the general index as an alternative to database methods that currently dominate?

finally, using something like the general index would permit search for features that were not discussed in the abstract (e.g. statistical tests used)

**Comment by Rensvandeschoot** [comment]
You might be able to make this database available like we did with the CORD19 database as we did via a [plug-in](https://github.com/asreview/asreview-covid19). 

 In [the documentation](https://asreview.readthedocs.io/en/latest/API/extension_dev.html), we provide more explanation about adding an extension. @JTeijema is currently updating the [new dataset template](https://github.com/asreview/template-extension-new-dataset) (see Issue https://github.com/asreview/asreview/discussions/1001).

**Reply by tsaltena** [reply]
Hi @bozo32,

We're working on enabling Semantic Search on top of OpenAlex. Currently we have a demo on a recent slice of the set (11 million works out of the 205 million they have), but we'd be very interested in verifying if using a Semantic Search approach would yield relevant papers not found through existing keyword searches. It's not trivial to scale up to the full set, but if you would be interested in running an experiment on a certain time slice of data please let me know!

**Comment by bozo32** [comment]
Hi

I have a group of 7-10 PhDs from a diversity of disciplines starting a cycle of systematic(ish) review who will be doing serious information retrieval in less than a month here at WUR. I say systematic(ish) because most of the PhDs are working in fields where reporting and research practice do not meet the conditions required for adequate identification and then meta-analysis of studies...and many of them have questions that require systematic information retrieval such as 'how do researchers operationalise (fancy construct)' which don't fit the model of classical SR. 

I could offer up a part of that cohort as guinea-pigs. 
I would select them based on the resolution with which they have been able to formulate their research interests (well formed is better) and for heterogeneity (I expect that there will be variations across academic disciplines).

To make things quite specific, they will be spending the day on 13 April trying to convert their research quesions into search strategies. That is when we'd have an opportunity to try and convert their interests into something that can be used.

I'm not sure how well openalex does correcting for the poor registration of indexed fields (abstract, title, keywords) and record content. The conventions and terms that find their ways into indexed fields seem, at times, to have an arbitrary relationship to the contents of the record they purport to summarise.  I was dearly hoping that something like the generalindex would permit direct query of some reasonable proxy of full text records.

I mentioned the possibility of some application of your work in improving the information retrieval stage of review to our Library here this morning. 

They were very interested.

-Peter
(Peter.tamas@wur.nl)
